classes/code/classes.py

```diff
@@ -0,0 +1,144 @@
+import webbrowser
+import matplotlib.pyplot as plt
+from Bio.Seq import Seq
+from Bio.SeqUtils import GC
+
+
+class RomaToday:
+    """
+    Parameters: 
+    
+    mood: good / bad / angry
+    lecturer: No / Yes
+    Location: any
+    
+    """
+
+    def __init__(self, mood="good", lecturer="No", location="DAS"):
+
+        self.mood = mood
+        self.lecturer = lecturer
+        self.location = location
+
+    def play_xbox(self):
+
+        if self.location == "DAS":
+            return "Game pass is my LOVE!"
+        return "I don't have Xbox here, cannot play directly :("
+
+    def help_students(self):
+
+        if self.mood == "good":
+            webbrowser.open('https://www.youtube.com/watch?v=NMbNwMKarcA')
+        elif self.mood == "bad":
+            webbrowser.open('https://www.youtube.com/playlist?list=PLjKdf6AHvR-EVtt5ocCAfU2WfJFobZzsD')
+        elif self.mood == "angry":
+            webbrowser.open('https://www.google.ru')
+
+    def drink_beer(self):
+        if self.lecturer == "No":
+            return True
+        elif self.lecturer == "Yes":
+            return False
+
+
+class RNA:
+
+    def __init__(self, sequence):
+        self.sequence = Seq(sequence)
+
+    def translation(self):
+        return self.sequence.translate()
+
+    def reverse_transcription(self):
+        return self.sequence.back_transcribe()
+
+
+class PlusSet(set):
+
+    def __init__(self, *args):
+        self.elements = [arg for arg in args if arg > 0]
```

Comment:
Сохранять это в атрибут совершенно не нужно
```suggestion
        elements = [arg for arg in args if arg > 0]
```